smoke-tests/apps/SamplingOverrides/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/ExceptionWithSamplingOverridesTest.java

```diff
@@ -19,45 +19,45 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-@UseAgent("applicationinsights3.json")
-abstract class ExceptionSamplingOverridesTest {
+@UseAgent("applicationinsights-exception-with-sampling-overrides.json")
+abstract class ExceptionWithSamplingOverridesTest {
 
   @RegisterExtension static final SmokeTestExtension testing = SmokeTestExtension.create();
 
   @Test
   @TargetUri(value = "/trackException")
-  void testExceptionSamplingOverrides() throws Exception {
+  void testExceptionWithSamplingOverrides() throws Exception {
     List<Envelope> rdList = testing.mockedIngestion.waitForItems("RequestData", 1);
     Envelope rdEnvelope = rdList.get(0);
     assertThat(rdEnvelope.getTags().get("ai.operation.name"))
         .isEqualTo("GET /SamplingOverrides/trackException");
-    assertThat(testing.mockedIngestion.getCountForType("ExceptionData")).isZero();
+    assertThat(testing.mockedIngestion.getCountForType("ExceptionData")).isEqualTo(1);
```

Comment:
don't we want zero exceptions here to show that the sampling overrides are working?